examples/benchmarks/odspsnapshotfetch-perftestapp/tsconfig.json

```diff
@@ -1,7 +1,9 @@
 {
-	"extends": "@fluidframework/build-common/ts-common-config.json",
+	"extends": [
+		"../../../common/build/build-common/tsconfig.base.json",
+		"../../../common/build/build-common/tsconfig.cjs.json",
+	],
 	"compilerOptions": {
-		"module": "esnext",
 		"rootDir": "./src",
 		"outDir": "lib",
```

Comment:
@tylerbutler, @jason-ha - Is this the right way to convert this consumer to node16?

Reply:
Not quite - see other comment. It's the pattern we currently use, though I think Jason has some ideas for simplifications/improvements. I still suggest following this same pattern; better to be consistent at the moment IMO.